packages/react/src/BlockSideMenu/components/DefaultButtons/BlockColorsButton.tsx

```diff
@@ -0,0 +1,80 @@
+import { ReactNode, useCallback, useRef, useState } from "react";
+import { Box, Menu } from "@mantine/core";
+import { BlockNoteEditor } from "@blocknote/core";
+import { HiChevronRight } from "react-icons/hi";
+import { ColorPicker } from "../../../SharedComponents/ColorPicker/components/ColorPicker";
+
+export const BlockColorsButton = (props: {
```

Comment:
I think this is the old code. Have you reviewed my changes / fixes to the color menu? (see PR)

Reply:
Are you sure? I looked at it again and it seems the same as in commit [30018f5](https://github.com/TypeCellOS/BlockNote/pull/143/commits/30018f50ad930adb5c518308a654b7f1b9af90dd), besides the functions for getting/setting colors, which changed as the props are different.

Reply:
You're right!